gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -98,11 +101,14 @@ def __repr__(self):
 
   def _values_asdict(self, header):
     """Converts PyVCF header values to dictionaries."""
+    ordered_dict = OrderedDict()
```

Comment:
Can you please elaborate why the order is important? It seems non-trivial to me and if there is a specific reason probably it should be documented either in this function's docstring or (better) in __init__.

Reply:
Sure, added a comment in `__init__`. Essentially, we get ordered dicts from pyvcf and we should continue to keep the order. It's important for fields like contigs and for cases where we want to go from VCF->VCF. Nuclues' [proto](https://github.com/google/nucleus/blob/master/nucleus/protos/variants.proto#L161) is also ordered for the same reason.

Reply:
Good point about merging: that's a different beast that we need to tackle when we have VCF->VCF. For now, we have a reasonable logic to continue to keep the order if no new fields are added. However, the semantics of VcfHeader should have order and it would pave the way towards Nucleus support.
RE asserts: that is a particular test where we're inferring headers, hence no predefined order. We ensure order is kept in all other tests.